src/introduction/qemu.md

```diff
@@ -0,0 +1,33 @@
+# Running QEMU
+
+This section is optional; some users of this guide will have been provided
+environments for building and running CHERI systems and programs.  If you have
+not, or if you also wish to be able to run things locally or in emulation,
```

Comment:
"locally or in emulation"? What is the "locally" meant to add here, beyond being incorrect in the case of Morello, given people have local Morello boards?